Tasks/PackerBuildV0/task.json

```diff
@@ -56,6 +56,22 @@
             },
             "defaultValue": "builtin"
         },
+        {
+            "name": "isManagedImage",
+            "type": "boolean",
+            "label": "Managed VM disk image",
+            "required": true,
+            "helpMarkDown": "Check if generated image is a managed image.",
```

Comment:
"Check if generated image should be a managed image."

Reply:
done